### PR TITLE
Feature uiscss code expectations pull-review

### PR DESCRIPTION
**Pull request by SotoArmando**
Hi all hope you good, this is a procedure pull-request. 

Looking forward to meet your expectations. 🚀 

Armando Soto

### CODE REVIEW

**Comment by SotoArmando** [comment]
[Demo link] https://mealsdbcapstoned.herokuapp.com/

Please avoid using unnecessary eslint-disable. This is a basic Microverse requirement. 

I used eslint-disable in order to avoid rules that were in conflict with scss functions and sizes modulation. since is main purpose of the code.

Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 10 minutes. The content of the video should include:

Please tell me why you submitted this project after the deadline passed. And tell me is student success know this? Yes they do i think so. All work was done in between friday and sunday was my misbehavior that caused that in first commit before my first day on work with this.


- [x] 5. We should not use global selectors like document.querySelector() or document.getElementById() in react


Your react app may still work while having these global selectors, but it is considered a top-rated bad practice.

- [x] 6. improve the project structure such that components for stateless components and a container/page folder for other components that use the state.

- [x] 8. We need to write more tests and fix failing tests, you need to write tests for all the modules in the App, it could be unit or integration testing.

In other words,

you need to test 80% of your methods. Make sure while testing you include both positive and negative scenarios. 

You need to write a test for the main logic of the app with unit or integration tests.

- [x] 9. Kindly work on the details page, work on the design.

- [x] 10. Make sure you work within the stipulated time, as developer meeting deadlines are going to be the order of the day. 

- [x] 11. Kindly fix JSX errors or warnings in the react console.
